extensions/Push/lib/Connector/Phabricator.pm

```diff
@@ -55,12 +54,40 @@ sub send {
     my $logger = Bugzilla->push_ext->logger;
 
     my $data = $message->payload_decoded;
-    my $bug_data = $self->_get_bug_data($data) || return 0;
-    my $bug = Bugzilla::Bug->new( { id => $bug_data->{id}, cache => 1 } );
+    my $bug = $self->_get_bug_by_data($data) || return PUSH_RESULT_OK;
+
+    # Q: Should we check to see if a group we don't care about changed?
+    # If so, then this doesn't automatically mean we need to make req to Phab?
+    # ======================================================
+    $is_public = is_public($bug);
```

Comment:
Well we can go down the path of checking the changes list and parse out the groups that have been added/removed and see if any of them belong to the sync groups. But I am not sure it is worth the extra complexity when we can just create a new policy. I am not worried about putting too many bogus policy entries in the database as this type of operation does not occur in practice.